File: Components/RewriteMe.Domain/Transcription/BillingPurchase.cs
﻿using System;

namespace RewriteMe.Domain.Transcription
{
    public class BillingPurchase
    {
        public Guid Id { get; set; }

        public Guid UserId { get; set; }

        public string PurchaseId { get; set; }

        public string ProductId { get; set; }

        public bool AutoRenewing { get; set; }

        public string PurchaseToken { get; set; }

        public string PurchaseState { get; set; }

        public string ConsumptionState { get; set; }

        public string Platform { get; set; }

        public DateTime TransactionDateUtc { get; set; }
    }
}

File: WebControlPanelClient/src/app/_models/billing-purchase.ts
export class BillingPurchase {
    id: string;
    userId: string;
    purchaseId: string;
    productId: string;
    autoRenewing: string;
    purchaseToken: string;
    purchaseState: string;
    consumptionState: string;
    platform: string;
    transactionDateUtc: Date;
}
